util/fipstools/inject_hash/inject_hash.go

```diff
@@ -357,7 +357,7 @@ func do(outPath, oInput string, arInput string, useSHA256 bool, macOS bool) erro
 
 	var moduleText, moduleROData []byte
 	var err error
-	if macOS == true {
+	if appleOS == true {
 		moduleText, moduleROData, err = doMacOS(objectBytes)
```

Comment:
minor nit: should we change to `doAppleOS`

Reply:
yup, changed